src/overlays/actors/ovl_Bg_Ydan_Hasi/z_bg_ydan_hasi.c

```diff
@@ -21,23 +32,179 @@ const ActorInit Bg_Ydan_Hasi_InitVars = {
     (ActorFunc)BgYdanHasi_Update,
     (ActorFunc)BgYdanHasi_Draw,
 };
-*/
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_Bg_Ydan_Hasi/BgYdanHasi_Init.s")
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_Bg_Ydan_Hasi/BgYdanHasi_Destroy.s")
+static InitChainEntry sInitChain[] = {
+    ICHAIN_VEC3F_DIV1000(scale, 100, ICHAIN_STOP),
+};
+
+Gfx* D_808BEC24[] = { 0x06007508, 0x06005DE0, 0x06005018 };
+
+extern UNK_TYPE D_06005780;
+extern UNK_TYPE D_06007798;
+Gfx* D_06005DE0[];
```

Comment:
This declaration should have an extern since it's not from this file. I don't even know how why the compiler is fine without it tbh, that's really confusing.